CHANGELOG.rst

```diff
@@ -5,6 +5,13 @@ Changelog History
 xskillscore v0.0.14 (2020-XX-XX)
```

Comment:
Mind rebasing? Apologies I released yesterday (https://github.com/raybellwaves/xskillscore/blob/master/CHANGELOG.rst)
You can create a 0.0.15

Reply:
May also have to update https://github.com/raybellwaves/xskillscore/blob/master/setup.py

Reply:
now setup.py wont have `VERSION=` but uses setuptools. therefore you need to run this publish command under `.github` after the merge to master. I think this only works from your system.